_unit-test/create-docker-volumes-test.sh

```diff
@@ -15,14 +15,13 @@ sentry-kafka
 sentry-postgres
 sentry-redis
 sentry-symbolicator
+sentry-vroom
 sentry-zookeeper"
 
 before=$(get_volumes)
 
 test "$before" == "" || test "$before" == "$expected_volumes"
 
-source install/create-docker-volumes.sh
```

Comment:
not sure why this was ever duplicated even after looking through the file history, and tests seem to be passing fine without it

Reply:
maybe testing for duplicate volume creation?

Reply:
Yes, this was to make sure trying to create volumes over and over again does not error. I think, since we run the whole installation twice now, this is redundant.